opencensus-shim/src/main/java/io/opentelemetry/opencensusshim/SpanConverter.java

```diff
@@ -50,6 +41,12 @@ protected FakeSpan(SpanContext context) {
       super(context, RECORD_EVENTS_SPAN_OPTIONS);
     }
 
+    @Override
```

Comment:
Do we still need FakeSpan? Can't we use one of the new Span implementations where this is being used?